uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.js

```diff
@@ -50,14 +50,20 @@ define([
         pending_requests: false,
         contact_id: null,
         selectedPeriod: null,
-        selectedAbsenceTypes: null
+        selectedAbsenceTypes: null,
+        assignedTo: 'all'
       }
     };
     vm.filteredUsers = [];
+    vm.filtersByAssignee = [
+      { type: 'all', label: 'All' },
+      { type: 'unassigned', label: 'Unassigned' },
+      { type: 'me', label: 'Assigned To Me' }
+    ];
     vm.isFilterExpanded = false;
+    vm.isAdmin = false; // this property is updated on controller initialization
```

Comment:
Right! However, it's better to keep the assignments grouped appropriately as I suggested . What's your opinion on this?